internal/cloud_provider/resource_cloud_provider.go

```diff
@@ -279,7 +274,7 @@ func resourceCloudProviderDiff() schema.CustomizeDiffFunc {
 
 				// if not IAM AWS cloud provider, check for credentials in env
 				if !isIAM {
-					_, isPresentAccessKeyID := os.LookupEnv(utils.AWSAccessKeyEnv)
+					_, isPresentAccessKeyID := os.LookupEnv(utils.GCPCredentialsEnv)
```

Comment:
This would still be AWSAccessKeyEnv since code == 'aws'?

Reply:
Fixed